### ISSUE TITLE
[java] UnusedImports false positive for static import

### ISSUE THREAD

**Issue by AdamCarroll** [pmd/pmd#925]
**Affects PMD Version:** 

6.0.1

**Rule:**

UnusedImports

**Description:**

A false positive is generated for the `UnusedImports` rule on static import of constants.

**Code Sample demonstrating the issue:**

The file [`FileFormatType`](https://github.com/VocabHunter/VocabHunter/blob/dea49f975484a0483fa5d7ff8fb368dd7a2ce347/gui/src/main/java/io/github/vocabhunter/gui/dialogues/FileFormatType.java) on the project [VocabHunter](https://github.com/VocabHunter/VocabHunter) shows this problem.  To work around the problem, the following line has been added to [ruleset.xml](https://github.com/VocabHunter/VocabHunter/blob/d2f2e536ab0aec1a17063c8dde1365006d39706e/config/pmd/ruleset.xml):
```
<exclude-pattern>.*/*FileFormatType.java</exclude-pattern>
```

To reproduce the problem, remove the above workaround line and then launch the following:
```
$ ./gradlew clean :gui:pmdMain
```

**Running PMD through:** *Gradle*

**Comment by adangel** [comment]
@AdamCarroll Thanks for reporting.
On first sight, it seems, that ondemand static imports are not handled correctly. That would be definitely a bug.

**Comment by AdamCarroll** [comment]
OK, thanks @adangel .  Let me know if you need any more information.

**Comment by adangel** [comment]
It turned out, we didn't consider package-private fields yet. This will be fixed with PMD 6.1.0.

**Comment by AdamCarroll** [comment]
OK great.  Thanks @adangel
